targets/LiteOS-stm32f4/source/main.c

```diff
@@ -0,0 +1,150 @@
+#include "los_sys.h"
```

Comment:
Related comment to that of @zherczeg's: are there any substantial portions of source code in this file that have been copy-pasted from existing sources? If so, do they need the reproduction of any copyright notices or list of conditions, and are they compatible with Apache 2.0 and our contribution guidelines?